### PR TITLE
Bootstrapper improvements

### PR DESCRIPTION
**Pull request by 0xnogo**

### CODE REVIEW

File: core/capabilities/ccip/oraclecreator/bootstrap.go [hunk 1]
```diff
@@ -37,13 +40,91 @@ import (
 
 var _ cctypes.OracleCreator = &bootstrapOracleCreator{}
 
+// bootstrapOracle wraps a CCIPOracle (the bootstrapper) and manages RMN-specific resources
+type bootstrapOracle struct {
+	baseOracle      cctypes.CCIPOracle
+	peerGroupDialer *peerGroupDialer
+	rmnHomeReader   ccipreaderpkg.RMNHome
+	started         bool
```
Comment (makramkd) [inline core/capabilities/ccip/oraclecreator/bootstrap.go]: Is this necessary? I think with the current lifecycle, once `Close` is called the object should be discarded forever.

File: core/capabilities/ccip/oraclecreator/bootstrap.go [hunk 2]
```diff
@@ -306,61 +484,6 @@ func (d *peerGroupDialer) closeExistingPeerGroups() {
...
-		lggr.Infow("Created new peer group successfully")
-
-		d.activePeerGroups = append(d.activePeerGroups, peerGroup)
-		d.activeConfigDigests = append(d.activeConfigDigests, genericEndpointConfigDigest)
-	}
-
-	return nil
-}
-
 func writePrefix(prefix ocr2types.ConfigDigestPrefix, hash cciptypes.Bytes32) cciptypes.Bytes32 {
```
Comment (makramkd) [inline core/capabilities/ccip/oraclecreator/bootstrap.go:475]: Lets create a ticket to extract the common pieces of this code and the code in chainlink-ccip and re-use where possible.

File: core/capabilities/ccip/oraclecreator/bootstrap.go (hunk 1 above)
Comment (0xnogo) [reply core/capabilities/ccip/oraclecreator/bootstrap.go]: This is also checking if the oracle has already been started. But I think we can remove it this logic as start should not be called more than once anyway.

File: core/capabilities/ccip/oraclecreator/bootstrap.go (hunk 2 above)
Comment (0xnogo) [reply core/capabilities/ccip/oraclecreator/bootstrap.go:475]: done!

**Pull request closed without merging**